key,default_value
userAgent,"python-requests/2.9.2"
eventID,"fcd10d8a-8bea-4720-85e7-52d7b362787"
errorMessage,"User: arn:aws:iam::811596193553:user/Level6 is not authorized to perform: dynamodb:CreateTable on resource: arn:aws:dynamodb:us-west-2:811596193553:table/TestTable"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDADO2GQD0K8TEF7KW1V"
userIdentity_arn,"arn:aws:iam::811596193553:user/Level6"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA3Z2XBVUDFQ9TU4MD"
userIdentity_userName,"Level6"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"31.2.9.0"
eventName,"CreateTable"
eventSource,"dynamodb.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-west-2"
requestID,"EN886V7VUQNC6PK305OH9R60UJVV4KQNSO5AEMVJF66Q9ASUAAJG"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2017-06-21T06:20:07Z"
